staging/src/k8s.io/kubectl/pkg/polymorphichelpers/helpers.go

```diff
@@ -80,6 +81,22 @@ func GetFirstPod(client coreclient.PodsGetter, namespace string, selector string
 	return pod, 1, nil
 }
 
+// GetPodCondition returns the condition for the passed in pod and condition type.
+// It returns an error if the condition is not found.
+func GetPodCondition(pod *corev1.Pod, condition corev1.PodConditionType) (*corev1.PodCondition, error) {
```

Comment:
I think we can use utility functions in here https://github.com/kubernetes/kubernetes/blob/d35da348c60a3c7505419741f2546ff8b0e38454/staging/src/k8s.io/kubectl/pkg/util/podutils/podutils.go#L46-L84 instead adding a new one?

Reply:
Ah good find. I figured this existed somewhere but couldn't find it.